### DISCUSSION TITLE
Can we set the autoRemove for Pre-built Filter Functions and import MRT_FilterFns?

### DISCUSSION THREAD

**Discussion by samchiu90** [KevinVandy/material-react-table#376] [Q&A] [answered]
In 
https://github.com/KevinVandy/material-react-table/discussions/223
and 
https://github.com/KevinVandy/material-react-table/blob/main/packages/material-react-table/src/filterFns.ts

We can see all built-in FilterFns will auto remove the filter when the value is falsy, such as 0 or false.
`(val: any) => !val;`

If I want to filter the column with value = false, (FilterFn).autoRemove should be something like  
`(val: any) => val === null || val === undefined || ( typeof val === 'string' &&  val.trim() === "" ) ; ` 

Otherwise, the filter will be removed after setting the filter value,
but it seems that we cannot set the `autoRemove` functions unless we set some custom filter functions to override the built-in FilterFns.

Then we need to import the MRT_FilterFns to customize their `autoRemove`, but `MRT_FilterFns` does not export for us to import.

I believe that we should be able to set the `autoRemove` in the table/column and import the `MRT_FilterFns` for us to customize.

**Comment by KevinVandy** [comment, marked as answer]
v1.8.0 just shipped and it includes exports for all MRT_FilterFns.

You can now import them, mutate them, and then pass them back to the MRT in the `filterFns` prop.
